Backoffice/LeftPanel/Strategies/ErrorPagesPanelStrategy.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\LeftPanel\Strategies;
+
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
+
+/**
+ * Class ErrorPagesPanelStrategy
+ */
+class ErrorPagesPanelStrategy extends AbstractLeftPanelStrategy
+{
+    const ROLE_ACCESS_TREE_NODE = 'ROLE_ACCESS_TREE_NODE';
```

Comment:
As the string it refers to is the same, i think that you can refer to another strategy.

Or you could change the name ROLE_ACCESS_TREE_NODE to ROLE_ACCESS_ERROR_TREE_NODE